src/css/variables.module.css

```diff
@@ -4,6 +4,29 @@
   };
 }
 
+html[data-theme='dark'] {
+  --accent-highlight: #6889E5;
+  --default-background-color: var(--dark-mode-black-darkest);
+  --default-border-color: var(--dark-mode-black-lighter);
+  --doc-component-background: var(--dark-mode-black-lighter);
+  --doc-primary: white;
+  --doc-title: white;
+  --nav-page-indicator-primary: white;
+  --nav-page-indicator-secondary: var(--dark-mode-gray-1);
+  --nav-primary-text-color: white;
+  --primary-gray: var(--dark-mode-gray-2);
+  --right-sidebar-top-bar-hover: var(--accent-highlight);
+  --sidebar-gray: var(--dark-mode-gray-1);
+  --sidebar-primary-title: var(--dark-mode-gray-2);
+  --snippet-copy-color: white;
+  --snippet-header-background: #50576B;
+  --subnav-background: #2A3045;
+  --subnav-text-color: var(--dark-mode-gray-1);
+  --toc-color: #A3ACB9;
+  --toggle-background: var(--dark-mode-gray-1);
+  --toggle-indicator-background: var(--default-background-color);
+}
+
```

Comment:
Changing these variables in this block enough to make all the changes for dark mode!